crates/cargo-plumbing-schemas/src/resolve_features.rs

```diff
@@ -63,6 +63,10 @@ pub enum ResolveFeaturesOut {
         #[serde(skip_serializing_if = "Vec::is_empty")]
         features: Vec<String>,
     },
+    Target {
+        name: String,
+        kind: String,
+    },
```

Comment:
> feat(resolve-features): Target selection 

The commit says its about target selection but this seems to be about reporting out what was selected